src/data/int/basic.lean

```diff
@@ -732,6 +735,13 @@ begin
     try {apply dvd_add}; assumption }
 end
 
+lemma sub_div_of_dvd {a b c : ℤ} (hca : c ∣ a) (hcb : c ∣ b) : (a - b) / c = a / c - b / c :=
```

Comment:
cf. [the new lemmas](https://github.com/leanprover-community/mathlib/pull/5752/files#diff-dbbbc0882e667430b70b0e868185c1d6bef472d12e5f849a2b7d40f18b1c3f3cR383-R395) added in #5752.